app/dashboard/templates/dashboard.html

```diff
@@ -145,7 +146,6 @@ <h3>{% trans "No results found." %}</h3>
     {% include 'shared/footer_scripts.html' %}
     {% include 'shared/rollbar.html' %}
     {% include 'shared/footer.html' %}
-
```

Comment:
Could we keep the extra line (serves as a distinction ) btwn the templates & scripts